src/app/components/ResultsList/ResultsList.jsx

```diff
@@ -103,7 +104,7 @@ const ResultsList = ({
         {
           hasRequestTable &&
           <ItemTable
-            items={items.slice(0, itemTableLimit)}
+            items={ItemSorter.sortItems(items.slice(0, itemTableLimit))}
```

Comment:
I assume `items` will generally include just the three (max) items fetched, so it makes no difference to apply the `slice` or the sort first, right? But say there are 2 checkin items and 3 items. You will end up with `items` being composed of 2 checkin items followed by the 3 real items in no defined order. If you `slice` that before applying the sort, you may get a different item in the third position than had you applied sort before the `slice`, right? I may have this wrong and this may be an unlikely scenario.

Reply:
Good point about this applying to checkin items. Since checkin items have their own sorting, I've moved the `ItemSorter` call to apply just to the regular items.

Regarding your comment above, the only reason I can see there would be some discrepancies is because there is some filtering going on. But I don't think this should cause discrepancies very often without any filters being applied.